### PR TITLE
Update check need uk visa

### PR DESCRIPTION
**Pull request by mgrassotti**
Trello: https://trello.com/c/oL55P4BR/792-update-check-if-you-need-a-uk-visa

1) Add an option for users in Macao and Hong Kong to choose whether they're travelling with a passport or travel document. If users are travelling with a travel document they'll need to be treated as the 'Visa_national' group in the logic.  

2) Remove a sentence from an outcome (Outcome 1: No visa needed).

[Preview (Hong Kong)](https://smart-answers-preview-pr-3316.herokuapp.com/check-uk-visa/y/hong-kong)
[Preview (Macao)](https://smart-answers-preview-pr-3316.herokuapp.com/check-uk-visa/y/macao)

### CODE REVIEW

File: test/integration/smart_answer_flows/check_uk_visa_test.rb [hunk 1]
```diff
@@ -662,6 +662,24 @@ class CheckUkVisaTest < ActiveSupport::TestCase
...
+      context '6 months or less' do
+        should "takes you to outcome_study_m" do
+          add_response "six_months_or_less"
+          assert_current_node :outcome_study_m
+        end
+      end
+      context 'more than 6 months' do
+        should "takes you to outcome_study_y" do
+          add_response "longer_than_six_months"
+          assert_current_node :outcome_study_y
```
Comment (elenatanasoiu) [inline test/integration/smart_answer_flows/check_uk_visa_test.rb]: Remarking on the commit message: 
```
If 6 months or less -> Outcome 3 Study M
If 6 months or more -> Outcome 3 Study Y
```
should say 
```
If 6 months or less -> Outcome 3 Study M
If 6 months or more -> Outcome 2 Study Y
```

Comment (suzannehamilton) [reply test/integration/smart_answer_flows/check_uk_visa_test.rb]: I've updated the commit message. 👍

**Comment by suzannehamilton** [comment]
I've added a [new commit](https://github.com/alphagov/smart-answers/pull/3316/commits/e05987133aca1b4647febfb9806a236647e8be6e) to this PR to split the flow for users with passports vs those with travel documents. Does anyone have time to review it?

**Comment by suzannehamilton** [comment]
This has been fact-checked, so it's good to merge.

**Pull request merged**